src/client/pfs/pfs.go

```diff
@@ -10,7 +10,7 @@ import (
 
 var (
 	// ChunkSize is the size of file chunks when resumable upload is used
-	ChunkSize = int64(16 * 1024 * 1024) // 16 MB
+	ChunkSize = int64(512 * 1024 * 1024) // 16 MB
```

Comment:
This change is unrelated to the FUSE changes but was something I noticed while testing it. This was causing us to have to store a ton of metadata when we inserted a large file. We were storing about 128 bytes per 16 MB in the file. Which was big enough to start to be a problem, particularly because it had to pass through etcd in many cases.

Will update the comment.